pom.xml

```diff
@@ -46,6 +51,8 @@
     </repository>
   </repositories>
 
+    
+    
   <pluginRepositories>
```

Comment:
generally don't stage unrelated modification (and especially formatting/whitespaces things).

Why: It's making future history lookup more difficult since your modification is gonna appear as last commit for those lines, when they actually weren't "actual" modifications.